### DISCUSSION TITLE
Goal 14: Conserve and sustainably use the oceans, seas and marine resources for sustainable development

### DISCUSSION THREAD

**Discussion by HSCunha** [Project-Resilience/platform#36] [General]
Chat about anything related to Goal 14: [Conserve and sustainably use the oceans, seas and marine resources for sustainable development](https://github.com/Project-Resilience/platform/blob/main/goals/goal_14.md)

**Comment by ilindholt** [comment]
### WhaleAgent: An open-source agentic solution to reduce whale strikes globally
**Background**: The change in the IMO decarbonisation plans, has refocused the industry on reducing the number of whale strikes. UN Sustainability Goal 14: to protect Marine Life.
**Key Problem**: Whale strikes are a leading cause of death for large whales, where the 20000 killed through vessel impact each year is threatening biodiversity and posing risks to the maritime industry. With 92% of global shipping routes overlapping habitats and traffic expected to triple by 2050
**The Challenge**: Accurately and rapidly predicting high-risk whale strike zones is extremely difficult due to sparse labelled data, the complexity of integrating diverse, time-sensitive environmental and vessel traffic data, and the lack of real-time predictive intelligence in current systems.
**The Solution**: To build an industry recognized open-source Global Adoption Roadmap, through the creation of research papers and an AI Prediction Framework for deployment that can tackle not only the reduction of whale strikes, but also the wider biodiversity impacts. The science world will need to be represented to validate the solution from a research perspective.
[WhaleAgent Executive Summary.pdf](https://github.com/user-attachments/files/23211117/WhaleAgent.Executive.Summary.pdf)

**Comment by somnrind** [comment]
**Stanford researchers find pattern in whale songs that predicts migration** 
https://news.stanford.edu/stories/2020/10/pattern-whale-songs-predicts-migration

**Comment by ofrancon** [comment]
**Smarter seas: Advancements in routing and voyage optimisation technology** 
- Combining weather, traffic, fuel, and performance data to generate efficient, real-time route plans 
- Using AI-driven predictive analytics to optimise voyages, reduce emissions, and minimise fuel costs 
- Accurately mirroring the actual performance of the ship as measured at sea

**Elevating vessel performance through data-driven risk & sustainability insights** 
- Leverage data-powered scoring, inspections, and vetting to pinpoint and proactively manage vessel risks 
- Benchmarking and monitoring greenhouse gas emissions, helping vessels align with sustainability goals 
- Streamline decision-making workflows for due diligence, improved transparency, and simplified compliance

**Performance vs emission: Can ships perform better while emitting less?** 
- Exploring the intricate relationship between ship performance and emissions 
- Unlocking opportunities to enhance efficiency, reduce costs, and control emissions 
- Delivering tailored strategies that align with your fleet’s performance goals

https://www.tech40.net/sbsm

**Comment by ofrancon** [comment]
Paper about whale migration patterns and the influence of environmental variables, providing insights for conservation efforts:
[Whisperers of whales wander: A directional statistical investigation of whales’ migration influenced by geomagnetic, ocean current, and celestial cues](https://www.sciencedirect.com/science/article/pii/S1617138125001888)

**Comment by ofrancon** [comment]
This paper describes possible interventions to reduce whale-ship collisions and their severity:
[Ship collision risk threatens whales across the world’s oceans](https://www.science.org/doi/10.1126/science.adp1950)
By combining whale locations and ships positions it's possible to create a whale-ship collision risk heatmap. The map is static for now but could include seasonality and additional factors.

Proposed interventions (actions) include:
  - reduce vessel speed in high risk zones
  - changes in vessel routes
  - expand protected areas

Outcomes can be measured in:
  - reduced number of collisions
  - lower severity of collisions

And also:
  - reduced air pollution
  - reduced green house gas emissions
  - reduced underwater noise pollution

Which would benefit marine species, the climate and public health
